products.php

```diff
@@ -0,0 +1,28 @@
+<?php
+header('Content-Type: application/json; charset=utf-8');
+header('Access-Control-Allow-Origin: *');
```

Comment:
this code is now duplicated in 2 files. perhaps put it into the response service or somewhere that makes it DRY